x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/saml/SamlObjectHandler.java

```diff
@@ -148,7 +148,11 @@ protected SpConfiguration getSpConfiguration() {
     }
 
     protected String describe(X509Certificate certificate) {
-        return "X509Certificate{Subject=" + certificate.getSubjectDN() + "; SerialNo=" + certificate.getSerialNumber().toString(16) + "}";
+        return "X509Certificate{Subject="
+            + certificate.getSubjectX500Principal()
```

Comment:
Hmm actually this one might be another breaking change; validating it

Reply:
Under the hood, both getSubjectX500Principal().toString() and getSubjectDN().toString() rely on X500Name.toString() so this should be fine

Reply:
The method is only used for logging messages which we don't provide any BWC.